src/mjolnir/ingest_transit.cc

```diff
@@ -87,20 +73,20 @@ template <> struct equal_to<feedObject> {
 namespace {
 
 struct tileTransitInfo {
-  GraphId graphId;
+  GraphId graphid;
   // TODO: unordered multimap-string to a pair of strings that maps from station (parent_id) ->
   //  platform/egress (child) (Max 2 kinds / many could exists)
-  std::unordered_multimap<feedObject, gtfs::Id> tile_station_children;
-  std::unordered_set<feedObject> tile_stops;
-  std::unordered_set<feedObject> tile_trips;
-  std::unordered_map<feedObject, size_t> tile_routes;
-  std::unordered_set<feedObject> tile_services;
-  std::unordered_set<feedObject> tile_agencies;
-  std::unordered_map<feedObject, size_t> tile_shapes;
+  std::unordered_multimap<feedObject, gtfs::Id> station_children;
+  std::unordered_set<feedObject> stations;
```

Comment:
this is really supposed to be stations, not generic stops, which go into the `station_children`.